indico/MaKaC/webinterface/tpls/HTMLHeader.tpl

```diff
@@ -22,12 +22,14 @@
 % endif
 
         <script type="text/javascript">
-                var TextRoot = "${ baseUrl }/js/indico/i18n/";
                 var ScriptRoot = "${ baseUrl }/js/";
         </script>
 
+        <!-- Jed-based i18n -->
+        <script type="text/javascript" src="${ url_for('assets.i18n_locale', locale_name=language) }"></script>
+
         <!-- Indico specific -->
-        ${ page._getJavaScriptInclude(str(urlHandlers.UHJSVars.getURL())) } <!-- Indico Variables -->
+        ${ page._getJavaScriptInclude(url_for('assets.js_vars_global')) } <!-- Indico Variables -->
```

Comment:
I'd already use a proper `<script>` tag (and that comment after it is a bit verbose.. someone who cares could simply look at the file)